blockchain/v0/reactor_test.go

```diff
@@ -289,7 +289,8 @@ func makeTxs(height int64) (txs []types.Tx) {
 }
 
 func makeBlock(height int64, state sm.State, lastCommit *types.Commit) *types.Block {
-	block, _ := state.MakeBlock(height, makeTxs(height), nil, nil, nil, lastCommit, state.Validators.GetProposer().Address)
+	block, _ := state.MakeBlock(height, makeTxs(height), nil,
+		nil, types.Messages{}, lastCommit, state.Validators.GetProposer().Address)
```

Comment:
Optional nit: I think, here and in other places, where the line becomes too large, we should rather use one line per argument. I think that improves the readability compared to breaking the line somewhere in the middle. 
Didn't this use to be convention in the tendermint code-base anyway?